src/invidious/jobs/blockage_status_job.cr

```diff
@@ -0,0 +1,51 @@
+class Invidious::Jobs::CheckBlockageStatusJob < Invidious::Jobs::BaseJob
+  private getter db : DB::Database
+
+  BLOCKAGE_STATUS = {
+    "version" => "1.0",
+    "blocked" => false,
+  }
+
+  def initialize(@db)
+  end
+
+  def begin
+    # Logic mostly taken from bypass_captcha_job.cr
+    loop do
+      begin
+        # TODO find performant way of fetching a random video from the videos table.
+        video = fetch_video("zj82_v2R6ts", nil)
```

Comment:
Usually we take the first youtube video, the zoo one.

Reply:
I just took the video from the bypass captcha job. I'd go ahead and switch to the zoo one